.pre-commit-config.yaml

```
@@ -30,7 +30,7 @@ repos:
     ]
 
 - repo: https://github.com/codespell-project/codespell
-  rev: v2.2.5
+  rev: v2.2.6
   hooks:
   - id: codespell
```

```suggestion
  - id: codespell
    args: [-w]
```

This will auto-fix errors, and then pre-commit will fix the PR.

---

Thank you.